debug/gdbdiff.ts

```diff
@@ -23,25 +67,29 @@ function compareRegisters(emulator: Uint32Array, silicone: Uint32Array) {
 async function main() {
   const client1 = new GDBClient();
   const client2 = new GDBClient();
-  await client1.connect('localhost');
-  await client2.connect('raspberrypi');
+  await client1.connect('localhost', 3334);
+  await client2.connect('localhost', 3333);
```

Comment:
I'm thinking, it may make more sense to call these `emulatorClient` and `siliconClient` instead of `client1`/`client2`. I know I chose those names, but now when I'm thinking of it, `client1` / `client2` are pretty bad names...